types/benchmark/benchmark-tests.ts

```diff
@@ -229,6 +229,27 @@ suite.add({
   'onComplete': onComplete
 });
 
+// unregister a listener for an event type
+suite.off('cycle', listener) as Benchmark.Suite;
```

Comment:
What are you hoping to test here with the cast?

Reply:
Just checking the return type.  Surely, it's obscure. Better code?

Reply:
I think that works but I'm not sure since it is a cast.

Reply:
Hmm, it's not for work, but for passing the compiler.
If you reverted it, this would cause the following errors.

```
benchmark-tests.ts(233,1): error TS2352: Type 'Benchmark' cannot be converted to type 'Suite'.
  Property 'length' is missing in type 'Benchmark'.
benchmark-tests.ts(236,1): error TS2352: Type 'Benchmark' cannot be converted to type 'Suite'.
benchmark-tests.ts(239,1): error TS2352: Type 'Benchmark' cannot be converted to type 'Suite'.
benchmark-tests.ts(242,1): error TS2352: Type 'Benchmark' cannot be converted to type 'Suite'.
benchmark-tests.ts(245,1): error TS2352: Type 'Benchmark' cannot be converted to type 'Suite'.
benchmark-tests.ts(248,1): error TS2352: Type 'Benchmark' cannot be converted to type 'Suite'.
benchmark-tests.ts(251,1): error TS2352: Type 'Benchmark' cannot be converted to type 'Suite'.
```